docs/en/latest/admin-api.md

```diff
@@ -23,6 +23,8 @@ title: Admin API
 
 ## Table of Contents
 
+- [Table of Contents](#table-of-contents)
```

Comment:
We can remove entire TOC

Reply:
all right, VSCode will automatically generate this TOC 🤔